scripts/run-artifactory.sh

```diff
@@ -1,42 +1,53 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
-export ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION:-7.37.14}
+source "${SCRIPT_DIR}/get-access-key.sh"
+export ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION:-7.37.15}
 echo "ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION}"
 
 set -euf
 
-docker-compose --project-directory ${SCRIPT_DIR} up -d --remove-orphans
+docker-compose --project-directory "${SCRIPT_DIR}" up -d --remove-orphans
+
+ARTIFACTORY_URL_1=http://localhost:8081
+ARTIFACTORY_URL_2=http://localhost:9081
 
 echo "Waiting for Artifactory 1 to start"
-until curl -sf -u admin:password http://localhost:8081/artifactory/api/system/licenses/; do
+until curl -sf -u admin:password ${ARTIFACTORY_URL_1}/artifactory/api/system/licenses/; do
     printf '.'
     sleep 4
 done
 echo ""
 
 echo "Waiting for Artifactory 2 to start"
-until curl -sf -u admin:password http://localhost:9081/artifactory/api/system/licenses/; do
+until curl -sf -u admin:password ${ARTIFACTORY_URL_2}/artifactory/api/system/licenses/; do
     printf '.'
     sleep 4
 done
 echo ""
 
 echo "Setting base URL for Artifactory 2. (Base URL for Artifactory 1 will be set by acceptance tests)"
-curl -X PUT http://localhost:9081/artifactory/api/system/configuration/baseUrl -d 'http://artifactory-2:8081' -u admin:password -H "Content-type: text/plain"
+curl -X PUT "${ARTIFACTORY_URL_2}/artifactory/api/system/configuration/baseUrl" -d 'http://artifactory-2:8081' -u admin:password -H "Content-type: text/plain"
 
 # docker cp doesn't support coping files between containers so copy to local disk first
-CONTAINER_ID_1=$(docker ps -q --filter ancestor=releases-docker.jfrog.io/jfrog/artifactory-pro:${ARTIFACTORY_VERSION} --filter publish=8080)
-CONTAINER_ID_2=$(docker ps -q --filter ancestor=releases-docker.jfrog.io/jfrog/artifactory-pro:${ARTIFACTORY_VERSION} --filter publish=9080)
+CONTAINER_ID_1=$(docker ps -q --filter "ancestor=releases-docker.jfrog.io/jfrog/artifactory-pro:${ARTIFACTORY_VERSION}" --filter publish=8080)
+CONTAINER_ID_2=$(docker ps -q --filter "ancestor=releases-docker.jfrog.io/jfrog/artifactory-pro:${ARTIFACTORY_VERSION}" --filter publish=9080)
 
 echo "Fetching root certificates"
-docker cp ${CONTAINER_ID_1}:/opt/jfrog/artifactory/var/etc/access/keys/root.crt ${SCRIPT_DIR}/artifactory-1.crt \
-  && chmod go+rw ${SCRIPT_DIR}/artifactory-1.crt
-docker cp ${CONTAINER_ID_2}:/opt/jfrog/artifactory/var/etc/access/keys/root.crt ${SCRIPT_DIR}/artifactory-2.crt \
-  && chmod go+rw ${SCRIPT_DIR}/artifactory-2.crt
+docker cp "${CONTAINER_ID_1}":/opt/jfrog/artifactory/var/etc/access/keys/root.crt "${SCRIPT_DIR}/artifactory-1.crt" \
+  && chmod go+rw "${SCRIPT_DIR}"/artifactory-1.crt
```

Comment:
that was @chb0github [recommendation](https://github.com/jfrog/terraform-provider-artifactory/pull/446/files#diff-53b8211992d390270d52c8f050b2cafe77ec6cfedaa1e2430ac9891d01bd42cd). It should work, but I can't fully test it, because of M1.

Reply:
Anything you use a substitution variable you really need to `""` the whole param to prevent globbing